src/Microsoft.ML.Data/Scorers/BinaryClassifierScorer.cs

```diff
@@ -13,6 +13,7 @@
 using Newtonsoft.Json.Linq;
 using Microsoft.ML.Runtime.Model.Pfa;
 using System.Collections.Generic;
+using Microsoft.ML.Data;
```

Comment:
>using [](start = 0, length = 5)

I'm sorting all the usings now, don't do the Latin counting again!

Reply:
I can do Japanese! Ichi, ni, san, shi ...

---
In reply to: [242338122](https://github.com/dotnet/machinelearning/pull/1893#discussion_r242338122) [](ancestors = 242338122)